ui/state/token.ts

```diff
@@ -81,3 +85,48 @@ export const walletTokenBalance = selectorFamily<
       return { balance, error: null }
     },
 })
+
+export const nativeWalletTokenBalance = selectorFamily<
+  TokenBalanceResponse,
+  string | undefined | null
+>({
+  key: "nativeWalletTokenBalance",
+  get:
+    (tokenDenom) =>
+    async ({ get }) => {
+      if (!tokenDenom) return { balance: null, error: null }
```

Comment:
What's the point of allowing this to be null if we just pass it through?

Reply:
Hooks can't be conditionally called in components, so the alternative would be passing an empty string to the selector and running more code (the few lines below this that try to find a matching payToken) before just passing null through anyways.